Sofa/Component/LinearSystem/src/sofa/component/linearsystem/MatrixLinearSystem.h

```diff
@@ -80,6 +81,7 @@ class MatrixLinearSystem : public TypedMatrixLinearSystem<TMatrix, TVector>
     Data< bool > d_applyProjectiveConstraints; ///< If true, projective constraints are applied on the global matrix
     Data< bool > d_applyMappedComponents; ///< If true, mapped components contribute to the global matrix
     Data< bool > d_checkIndices; ///< If true, indices are verified before being added in to the global matrix, favoring security over speed
+    Data< bool > d_parallelAssemblyIndependantMatrices; ///< If true, independant matrices (global matrix vs mapped matrices) are assembled in parallel
```

Comment:
```suggestion
    Data< bool > d_parallelAssemblyIndependentMatrices; ///< If true, independent matrices (global matrix vs mapped matrices) are assembled in parallel
```

https://en.wiktionary.org/wiki/independant 🙊